contracts/payment/DelegateManager.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Please use the same header as the other contracts in this repository. You should also install solidity in code and format the files.

Reply:
@ChrisLahaye by "same header" you meant it needs a bank line between SPDX and pragma?